public/app/features/search/page/components/ManageActions.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Button, Checkbox, HorizontalGroup, useStyles2 } from '@grafana/ui';
+
+import { getStyles } from './ActionRow';
+
+type Props = {
+  items: Map<string, Set<string>>;
+};
+
+export function ManageActions({ items }: Props) {
```

Comment:
@axelavargas -- can you replace this with a version that does more than TODO :)

Reply:
If this PR looks reasonable enough, can you merge it and then start your own to implement MOVE/DELETE?  or you can push to this same branch, but I would like to merge ASAP since I am also working on the hierarchy and backend searcher implementation.